app/controllers/admin/statistics_announcement_publications_controller.rb

```diff
@@ -0,0 +1,61 @@
+class Admin::StatisticsAnnouncementPublicationsController < Admin::BaseController
+  before_action :find_statistics_announcement
+  layout "design_system"
+
+  def index
+    if params[:search].present?
```

Comment:
If we do this then this would need be updated to `params[:title]` https://github.com/alphagov/whitehall/pull/7660/files#r1195196818

Reply:
✅